app/src/main/java/org/simple/clinic/patientattribute/entry/BMIEntryEffect.kt

```diff
@@ -9,3 +9,6 @@ data class CreateNewBMIEntry(
     val patientUUID: UUID,
     val reading: BMIReading
 ) : BMIEntryEffect()
+sealed class BMIEntryViewEffect : BMIEntryEffect()
```

Comment:
Add a line break above this please?

Reply:
Ah, I was reviewing it per commit. Looks like it was broken in the initial commit and resolve later. My bad.